channels/channel.be/vier/chn_vier.py

```diff
@@ -570,6 +570,12 @@ def update_video_item_with_id(self, item: MediaItem) -> MediaItem:
             if error_message == "Locked":
                 # set it for the error statistics
                 item.isGeoLocked = True
+
+                whatsonId = item.metaData.get("whatsonId", None)
+                if whatsonId:
+                    # 2615619 = GoPlay contentSourceID.
+                    return self.__get_ssai_streams(item, 2615619, whatsonId, None)
```

Comment:
`self.__get_ssai_streams` heeft maar 2 parameters? Is `2615619` niet dynamisch ergens uit halen?

Reply:
Er zit nog de drm_header bij. 2615619 wordt teruggegeven bij de long form url, ik heb nog niet een locatie gevonden waar dit anders in zit, maar ze gebruiken dit nummer al meer dan een jaar van wat ik zo 123 terug kon vinden.